draft-dekater-scion-dataplane.md

```diff
@@ -1445,24 +1442,25 @@ SCION attempts to take the IANA's assigned Internet protocol numbers into consid
 
 The protocol numbers are used in the SCION header to identify the next level protocol.
 
+SCION reserves a number for a future protocol: the SCION Control Message Protocol (SCMP). Support for this protocol is optional and it has not yet been formally specified. A work-in-progress specification exists: [the scion.org SCMP specification document](https://docs.scion.org/en/latest/protocols/scmp.html).
```

Comment:
Maybe we could remove this comment and leave it in the table? I think we have the same problem for DrKey somewhere else

Reply:
I was acting on Eliot's suggestion: mention the protocol informally. If we just leave in the table with no explanation, we *will* get the question "what is this?". What is the motivation for your suggestion?